src/Microsoft.AspNet.Mvc.Core/Controller.cs

```diff
@@ -1026,6 +1026,57 @@ public virtual Task<bool> TryUpdateModelAsync<TModel>([NotNull] TModel model)
                 predicate);
         }
 
+        /// <summary>
+        /// Validates the specified <paramref name="model"/> instance.
+        /// </summary>
+        /// <param name="model">The model to validate.</param>
+        /// <returns>A <see cref="bool"/> that on completion returns <c>true</c> if the <see cref="ModelState"/>
```

Comment:
Is there a reason this PR comment is being ignored:

* https://github.com/aspnet/Mvc/pull/1807#discussion_r23041661 
* https://github.com/aspnet/Mvc/pull/1807#discussion_r23058006
* https://github.com/aspnet/Mvc/pull/1807#discussion_r23060229